### DISCUSSION TITLE
FindSubCluster() in Seurat v4 on integrated datasets

### DISCUSSION THREAD

**Discussion by camelest** [satijalab/seurat#4842] [Q&A]
Hi, first of all, thank you so much for this wonderful tool. 
I have a question regarding the new FindSubCluster() function.

I've integrated 2 datasets by SCTransform pipeline according to your vignette. After reading multiple posts about subclustering and DE analysis on integrated datasets (such as #1883 and #2812), which I assume happened before Seurat v4 was released, I have several questions as below: 

1. After all the discussion mentioned in the previous posts, is it better 
A) to go straight for integrated assay and run PCA etc. after subsetting or
B) to start from the raw data all over again, including "SCTransform"ing individual subsets and re-defining FindVariableFeatures() ?

2. In either case, what does FindSubCluster do? Does it consider features that was not included in the anchor-sets? 

3. Which assay in integrated analysis should we apply FindSubCluster to? According to #1717, I assume it's "integrated" assay, but it would be really nice if you could confirm it.

Thank you so much for your help.

**Comment by shaln** [comment]
I'd like to add an additional question that seem relevant to this post: Is there a way to plot the UMAP to show only the subclusters after FindSubCluster? So, not removing the other 'primary' clusters, but just to hide them so that they don't show up on the plot. This will make it easier to visually see the new subclusters, especially if there are many of them.
